README.md

```diff
@@ -18,12 +18,15 @@ GO111MODULE=on /usr/lib/go-1.12/bin/go install github.com/tranvictor/jarvis
 ## MacOS Build
 1. Download and install Go v1.12 [here](https://golang.org/dl/)
 ```
-go get github.com/tranvictor/jarvis
-go install github.com/tranvictor/jarvis
-cd jarvis
-go build -v
+GO111MODULE=on go get github.com/tranvictor/jarvis
+GO111MODULE=on go install github.com/tranvictor/jarvis
 ```
-2. Try `./jarvis --help`
+
+`jarvis` binary file will be placed at `$GOPATH/bin/`
+
+If the installation process is error, try to clear Go module cache at `$GOPATH/pkg/mod`
```

Comment:
should place it with "if the installation process returned errors" to have proper english.

Reply:
actually i think clear go module cache is not needed in the readme as every go package require to do this when it got dependencies version conflict

Reply:
@ductm54 it's ok i think, the purpose of the readme is to help users to use Jarvis, especially it is still very small and simple now, should be fine. if it ever grows to a huge readme, we can consider referring it to somewhere else.